src/modules/launcher/Plugins/Microsoft.Plugin.Folder/ContextMenuLoader.cs

```diff
@@ -40,7 +40,7 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
                     PluginName = Assembly.GetExecutingAssembly().GetName().Name,
                     Title = Properties.Resources.Microsoft_plugin_folder_copy_path,
                     Glyph = "\xE8C8",
-                    FontFamily = "Segoe MDL2 Assets",
+                    FontFamily = "Segoe Fluent Icons,Segoe MDL2 Assets",
```

Comment:
File _generic.xaml_ (from the WASDK 1.4 package) has the exact same content:
```xaml
            <FontFamily x:Key="SymbolThemeFontFamily">Segoe Fluent Icons,Segoe MDL2 Assets</FontFamily>
```
Can't we use that, in stead of hardcoding?

Reply:
This is WPF... not WinUI/WASDK. We could potentially default this FontFamily so that every plugin uses this, but I'd say that this is out of scope for this PR.